src/themes/masschallenge/modules/checkbox.overrides

```diff
@@ -0,0 +1,72 @@
+  ;
+}
+
+/* Checkmark */
+.ui.checkbox label:after,
+.ui.checkbox .box:after {
+  font-family: 'Checkbox';
+}
+
+/* Checked */
+.ui.checkbox input:checked ~ .box:after,
+.ui.checkbox input:checked ~ label:after {
+  content: '\e800';
+}
+
+/* Indeterminate */
+.ui.checkbox input:indeterminate ~ .box:after,
+.ui.checkbox input:indeterminate ~ label:after {
+  font-size: 12px;
+  content: '\e801';
+}
+
+
+/*  UTF Reference
+.check:before { content: '\e800'; }
+.dash:before  { content: '\e801'; }
+.plus:before { content: '\e802'; }
+*/
+
+/*******************************
+         MC Overrides
+*******************************/
+
+
+.ui.radio.checkbox.primary label {
+  color: @primaryColor;
+  font-weight: 600;
+  margin-right: 20px;
+  border: 7px solid @whiteColor;
+}
+
+.ui.radio.checkbox.primary label:hover {
+  color: @primaryColor;
+  background-color: @greyBackground;
+  border: 7px solid @greyBackground;
+  border-radius: 3px
+
+}
+
+.ui.radio.checkbox.default label {
+  margin-right: 20px;
+  border: 7px solid @whiteColor
+}
+
+.ui.radio.checkbox.default label:hover {
+  background-color: @greyBackground;
+  border: 7px solid @greyBackground;
+  border-radius: 3px
+
+}
```

Comment:
As written these changes will impact all radio buttons which I don't think is ideal. I'd suggest not making direct overrides and either make a new class or compounding existing classes that will have no unintended consequences. For example, instead of this override you could add the `button` class to the UI element and create this override (not really an override since there is no such thing as `.ui.radio.checkbox.button` in SUI to my knowledge.
```
.ui.radio.checkbox.button {
  background-color: transparent;
  padding: .5em 1em;
}
.ui.radio.checkbox.button:hover {
  background-color: #efebeb;
}
.ui.radio.checkbox.button label {
  font-weight: 600;
}
```
This produces the same result with less CSS because it leverages some of the shared styles for color and font that are available via the `button` class.

<img width="340" alt="Screen Shot 2020-05-19 at 9 23 31 AM" src="https://user-images.githubusercontent.com/11574697/82331659-6c64df80-99b2-11ea-95f9-47cde6305219.png">